core/pkg/opaprocessor/processorhandler.go

```diff
@@ -269,6 +260,20 @@ func (opap *OPAProcessor) processRule(ctx context.Context, rule *reporthandling.
 	return resources, nil
 }
 
+// appendPaths appends the failedPaths, fixPaths and fixCommand to the paths slice with the resourceID
+func appendPaths(paths []armotypes.PosturePaths, failedPaths []string, fixPaths []armotypes.FixPath, fixCommand string, resourceID string) []armotypes.PosturePaths {
```

Comment:
@YiscahLevySilas1  This is ok, I think we should have a "path" object and the "kind" of path, e.g.
```
{
    "type": "fix",
    "path": "/fixed/path",
    "resourceID": "id"
}
```
This way we can add different types, and in general it is scalable.